.vscode/settings.json

```diff
@@ -14,7 +14,7 @@
     "editor.tabSize": 2
   },
   "python.formatting.blackPath": "black",
-  "python.formatting.provider": "none",
+  "python.formatting.provider": "black",
   "python.formatting.blackArgs": [
```

Comment:
Removing these lines below on here will mean that formatOnSave starts working better